src/sentry/incidents/logic.py

```diff
@@ -781,5 +789,4 @@ def delete_snuba_subscription(subscription_id):
     :param subscription_id: The uuid of the subscription to delete
     :return:
     """
-    resp = safe_urlopen(settings.SENTRY_SNUBA + "/subscriptions/%s" % subscription_id, "DELETE")
-    resp.raise_for_status()
+    _snuba_pool.urlopen("DELETE", "/subscriptions/%s" % subscription_id, retries=False)
```

Comment:
Nit: `retries=False` here is redundant as retries only apply to `POST` and `GET`: https://github.com/getsentry/sentry/blob/95767d455b8004ec4b4c5026d84b64b6348e6d37/src/sentry/utils/snuba.py#L241

Reply:
We can/should add DELETE to that list too. I only added GET/POST becuase we didn't do any other operations.